.github/workflows/docker-builds.yml

```diff
@@ -65,20 +65,33 @@ jobs:
       - name: Build docker image
         id: build-docker-image
         uses: ./.github/actions/calculate-docker-image
-        env:
-          GHCR_PAT: ${{ secrets.GHCR_PAT }}
         with:
           docker-image-name: ${{ matrix.docker-image-name }}
           always-rebuild: true
           skip_push: false
           force_push: true
-          push-ghcr-image: ${{ github.event_name == 'push' }}
 
       - name: Pull docker image
         uses: pytorch/test-infra/.github/actions/pull-docker-image@main
         with:
           docker-image: ${{ steps.build-docker-image.outputs.docker-image }}
 
+      - name: Push to https://https://ghcr.io/
+        id: push-to-ghcr-io
+        if: ${{ github.event_name == 'push' }}
+        env:
+          ECR_DOCKER_IMAGE: ${{ steps.build-docker-imae.outputs.docker-image }}
+          GHCR_PAT: ${{ secrets.GHCR_PAT }}
+          IMAGE_NAME: ${{ matrix.docker-image-name }}
+        shell: bash
+        run: |
```

Comment:
You could stick this inside a retry action in case of future GH network issues

Reply:
Sure, though it might overload the system even more (I.e. I've retried it twice, and it failed twice...)